### PR TITLE
[WIP] fixes #17112 - pulp_docker_registry_port in katello.yaml

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

File: templates/katello.yaml.erb [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
   :post_sync_url: https://<%= scope['katello::fqdn'] %><%= scope['katello::deployment_url'] %>/api/v2/repositories/sync_complete?token=<%= scope['katello::post_sync_token'] %>
 
-  :pulp_docker_registry_port: <%= scope['katello::pulp_docker_registry_port'] %>
+  :pulp_docker_registry_port: <%= scope['katello::docker_registry_port'] %>
```
Comment (thomasmckay) [inline templates/katello.yaml.erb]: you are (were) correct! fixed

**Comment by thomasmckay** [comment]
This doesn't actually work. Passing --katello-docker-registry-port=5001 to installer still leaves katello.yaml at 5000

**Comment by ehelms** [comment]
I'm not sure why it wouldn't work, the code looks fine.

**Comment by ekohl** [comment]
Closing for inactivity.

**Pull request closed without merging**